plugin/mutate/source/dextool/plugin/mutate/backend/report/html/package.d

```diff
@@ -1522,8 +1510,26 @@ auto spawnOverviewActor(OverviewActor.Impl self, FlowControlActor.Address flowCt
         import dextool.plugin.mutate.backend.database.type : FileScore;
 
         auto fileScores = ctx.state.get.db.getMutationFileScoreHistory();
+
         ctx.state.get.files.toIndex(content, HtmlStyle.fileDir, fileScores);
 
+        //Change column: timeframe and value calculation handling
+        auto mut_data = appender!(string[])();
+        mut_data.put("var file_score_data = {};");
+
+        if (fileScores.length > 0) {
+            foreach (fileScoreKey; fileScores.byKey) {
+                mut_data.put(format("file_score_data['%s'] = {};", fileScoreKey));
+                foreach (score; fileScores[fileScoreKey]) {
+                    mut_data.put(format("file_score_data['%s']['%s'] = %f;",
+                            fileScoreKey, score.timeStamp, score.score.get));
+                }
+            }
+        }
+
+        auto script = index.root.childElements("head")[0].addChild("script");
```

Comment:
these two lines can be merged and the helper variable script can be removed.

```d
index.root.childElements("head")[0].addChild("script").appendChild(new RawSource(index, mut_data.data.joiner("\n").toUTF8));
```

something like that